xbmc/input/ButtonTranslator.cpp

```diff
@@ -1195,6 +1195,35 @@ CAction CButtonTranslator::GetGlobalAction(const CKey &key)
   return GetAction(-1, key, true);
 }
 
+bool CButtonTranslator::IsPossibleLongpress(int window, const CKey &key)
+{
+  map<int, buttonMap>::const_iterator it = m_translatorMap.find(window);
+  if (it == m_translatorMap.end())
+    return false;
+
+  uint32_t code = key.GetButtonCode();
+  code |= CKey::MODIFIER_LONG;
+  buttonMap::const_iterator it2 = (*it).second.find(code);
+
+  if (it2 != (*it).second.end())
+    return true;
+
+#ifdef TARGET_POSIX
+  // Some buttoncodes changed in Hardy
+  if ((code & KEY_VKEY) == KEY_VKEY && (code & 0x0F00))
+  {
+    code &= ~0x0F00;
+    it2 = (*it).second.find(code);
+    if (it2 != (*it).second.end())
+      return true;
+  }
+#endif
+  if (window > -1)
+    return IsPossibleLongpress(GetFallbackWindow(window), key);
```

Comment:
please add a comment that this will first check for a fallback window and if not found it'll check the global section of the keymap.